app/assets/stylesheets/darkswarm/_shop-filters.css.scss

```diff
@@ -156,6 +156,14 @@
       font-weight: bold;
       color: $teal-500;
     }
+
+    &.filter-label {
+      float: left;
+    }
+  }
+
+  @media all and (max-width: 1024px) {
```

Comment:
you can use the new breakpoint mixins?

Reply:
Ah yes, I rebased to pick up those changes but forgot to update this bit :+1:

Reply:
:+1: